Paganism/Interpreter/Data/Extensions/Extension.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Paganism.PParser.AST.Enums;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Paganism.PParser;
+using Paganism.PParser.AST;
+using Paganism.PParser.Values;
+using Paganism.Exceptions;
+using System.Collections.ObjectModel;
+
+#nullable enable
+namespace Paganism.Interpreter.Data.Extensions
+{
+    internal class Extension
+    {
+        public static List<string> AllowedExtensions { get; set; } = new()
+        {
+            "StringExtension"
+        };
+        public static Dictionary<TypesType, string> TypeExtensionAssociation { get; } = new()
+        {
+            { TypesType.String, "StringExtension" },
+            { TypesType.Char, "StringExtension" },
+            { TypesType.Number, "NumberExtension" },
+            { TypesType.Array, "ArrayExtension" }
+        };
```

Comment:
mmm.... maybe just string, number? Anda lso why char it is string?

Reply:
Better to handle a char like a string

Reply:
and for numbers can be cool do to things like
```cs
int Fox = 1.72812818
Fox.Round(1)
```